packages/fuels-test-helpers/src/node.rs

```diff
@@ -6,10 +6,11 @@ use std::{
     time::Duration,
 };
 
-use fuel_core_chain_config::{ChainConfig, StateConfig};
+//TODO: improve `ChainConfig` confiuration and integration with the provider
```

Comment:
@iqdecay Do you want to fix that in this pr or in the follow-up? Also, maybe it is worth adding more description of what exactly we want to improve=)

Reply:
I've tried this in #966 to see how it looks

Reply:
It looks kinda bad so I'm going to remove this comment and track how we can improve the UX in #975.